README.md

```diff
@@ -3,14 +3,49 @@ appium-idb
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/appium/appium-idb.svg)](https://greenkeeper.io/)
 
-Work in progress, stay tuned!
+appium-adb is NodeJS wrapper over iOS Device Bridge (idb) set of utilities made by Facebook. Read https://www.fbidb.io for more details.
+
+
+## Requirements
+
+- Python 3.6 or newer
+- XCode 10 or newer
+- idb_companion
+- idb
+
+
+## Installation
+
+Use the following commands to setup idb and its dependencies:
+
+`brew tap facebook/fb`
```

Comment:
This formats the three on a single line. Is that what is wanted?

<img width="764" alt="Screenshot 2019-05-22 14 30 32" src="https://user-images.githubusercontent.com/2614354/58199290-2b05fb00-7c9e-11e9-8847-5ac72d357f8a.png">

Reply:
fixed